mmdeploy/codebase/mmpose/models/backbones/litehrnet.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn.functional as F
+
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    'mmpose.models.backbones.litehrnet.CrossResolutionWeighting.forward')
+def cross_resolution_weighting__forward(ctx, self, x):
+    """Rewrite ``forward`` for default backend.
+
+    Rewrite this function to support export ``adaptive_avg_pool2d``.
+
+    Args:
+        x (list): block input.
+    """
+
+    mini_size = [int(_) for _ in x[-1].shape[-2:]]
+    out = [F.adaptive_avg_pool2d(s, mini_size) for s in x[:-1]] + [x[-1]]
```

Comment:
Would it be better to do the concate before adaptive_avg_pool2d?
We only need 1 pool2d node.

Reply:
x is a list of different w, h tensors and cannot be concated before pooling.